api/argo-templates/render_test.go

```diff
@@ -33,51 +33,29 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-func TestRenderRebuildTemplate(t *testing.T) {
-	t.Run("It should render a workflow template for a node", func(t *testing.T) {
-		// loop test 3 types: master/worker/storage
-		assert.Fail(t, "NOT IMPLEMENTED")
-	})
-
-	t.Run("It should fail when parameters are invalid", func(t *testing.T) {
-		// loop test: hostname, xname, image version
-		assert.Fail(t, "NOT IMPLEMENTED")
-	})
-}
-
-func TestRenderMasterRebuildTemplate(t *testing.T) {
-	t.Run("It should render a workflow template for a master node", func(t *testing.T) {
-		assert.Fail(t, "NOT IMPLEMENTED")
-	})
-
-	t.Run("It should fail when host is not a master node", func(t *testing.T) {
-		assert.Fail(t, "NOT IMPLEMENTED")
-	})
-}
-
 func TestRenderWorkerRebuildTemplate(t *testing.T) {
 	t.Run("It should render a workflow template for a worker node", func(t *testing.T) {
 		targetNcn := "ncn-w99999"
-		a, _ := GetWrokerRebuildWorkflow(targetNcn, "")
+		a, _ := GetWrokerRebuildWorkflow([]string{targetNcn}, "")
 		assert.Equal(t, true, strings.Contains(string(a), targetNcn))
 	})
 	t.Run("It should fail when host is not a worker node", func(t *testing.T) {
```

Comment:
I find this test a bit confusing, because it says 'it should fail', which gives the impression that the failures in the test should be 'false'. However, since it checks to see if the error exists, it's the opposite.

Reply:
i am going to add this to the render ticket and revisit this when i work on that